dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProcessTaskRelationMapper.xml

```diff
@@ -200,7 +200,11 @@
             and post_task_code = #{postTaskCode}
         </if>
     </select>
-
+    <select id="queryByWorkGroupName" resultType="org.apache.dolphinscheduler.dao.entity.ProcessTaskRelation">
+        select
+            relation.id
+        from t_ds_process_task_relation relation, t_ds_task_definition_log log where
+        log.code = relation.post_task_code and log.worker_group = #{workerGroupName}
```

Comment:
It looks miss `</select>`, Why not query `t_ds_task_definition_log` directly?

Reply:
When user delete workerGroup, is it better to check references in `t_ds_task_definition`, but not `t_ds_task_definition_log`? 
If just the old version taskDefinition has this references, can it allow to delete?